apps/src/templates/teacherHomepage/CollapsibleSection.jsx

```diff
@@ -72,13 +72,24 @@ const CollapsibleSection = React.createClass({
   renderContent() {
     if (this.state.open) {
       return (
-        <div style={styles.content}>
+        <div>
           {React.Children.map(this.props.children, (child, index) => {
-            return (
-              <div key={index}>
-                {child}
-              </div>
-            );
+            if (index % 2 === 0) {
+              return (
+                <div key={index}>
+                  {child}
+                    <div style={styles.spacer}>
```

Comment:
This works the way I want it to, but seems a little funky. Is there a better way to handle this?